docs/guide/acquiring_data/helioviewer.rst

```diff
@@ -20,6 +20,7 @@ We provide the follwing functions:
 2. As above but return the full JSON response instead of an image.
 3. Return all available datasources.
 4. Generates custom screenshots that allow labels and layers of images.
+5. Download the XML header embedded in a JPEG2000 image. This includes the FITS header as well as a section of Helioviewer-specific metadata.
```

Comment:
This should be reworded, I am not a fan of a copy and paste.

Reply:
Sure I will change that